### PR TITLE
Add some tests to blacklist in cache=none scenario for xfstest

### PR DESCRIPTION
**Pull request by SiboWang1997**
Some tests of xfstest not support mmap with cache=none.

ID: 1443

### CODE REVIEW

**Comment by SiboWang1997** [comment]
Hi @xiagao @zhenyzha ,  could you help review this patch when you are free? Thanks a lot.

File: qemu/tests/cfg/virtio_fs_share_data_multi_backend.cfg [hunk 1]
```diff
@@ -208,6 +208,13 @@
                         generic_blacklist += 'generic/070\ngeneric/650\n'
                     with_cache.always..with_nfs_source.multi_fs:
                         generic_blacklist += 'generic/451\n'
+                    # These tests not support mmap with cache=none
+                    with_cache.none..with_nfs_source.multi_fs:
```
Comment (zhenyzha) [inline qemu/tests/cfg/virtio_fs_share_data_multi_backend.cfg:212]: Should this condition be independent of `with_nfs_source.multi_fs`?
If there is no correlation, why don't we use the filter below?
`with_xfstest..with_cache.none:`

Comment (SiboWang1997) [reply qemu/tests/cfg/virtio_fs_share_data_multi_backend.cfg:212]: Yes, the `with_xfstest..with_cache.none` filter is more suitable, and I think the above filter (line 209) has the same problem，can I fix these two problems in this commit?

Comment (zhenyzha) [reply qemu/tests/cfg/virtio_fs_share_data_multi_backend.cfg:212]: If they are for the same reason, I think they can be merged together.

**Comment by xiagao** [comment]
It looks good to me.
Ack.

**Review by xiagao — APPROVED** [review]

**Review by zhenyzha — APPROVED** [review]

**Comment by SiboWang1997** [comment]
Hi @YongxueHong, could you help review this patch? Thanks a lot.

File: qemu/tests/cfg/virtio_fs_share_data.cfg [hunk 2]
```diff
@@ -296,6 +296,13 @@
                     generic_blacklist = 'generic/003\ngeneric/120\ngeneric/426\ngeneric/467\ngeneric/477\ngeneric/551\n'
                     with_xfstest..with_cache.always:
                         generic_blacklist += 'generic/451\n'
+                    # These tests not support mmap with cache=none, id2229115
```
Comment (YongxueHong) [inline qemu/tests/cfg/virtio_fs_share_data.cfg:299]: We don't recommend adding the internal information here since some upstream users can not access it for details, so we could summary it here.

File: qemu/tests/cfg/virtio_fs_share_data_multi_backend.cfg [hunk 3]
```diff
@@ -206,8 +206,15 @@
                     generic_blacklist = 'generic/003\ngeneric/120\ngeneric/426\ngeneric/467\ngeneric/477\ngeneric/551\ngeneric/035\ngeneric/531\n'
                     with_cache.auto.default_extra_parameters.default.with_nfs_source.multi_fs:
                         generic_blacklist += 'generic/070\ngeneric/650\n'
-                    with_cache.always..with_nfs_source.multi_fs:
+                    with_xfstest..with_cache.always:
```
Comment (YongxueHong) [inline qemu/tests/cfg/virtio_fs_share_data_multi_backend.cfg:209]: What's the reason for updating it?
Thanks.

**Comment by SiboWang1997** [comment]
Hi @YongxueHong, as we discussed, I have updated the patch, please help me review it, thanks a lot.

**Review by YongxueHong — APPROVED** [review]
LGTM.

**Comment by SiboWang1997** [comment]
python3 ConfigTest.py --guestname=RHEL.9.4.0..page_64k --platform=aarch64 --machines=arm64-pci --driveformat=virtio_scsi --nicmodel=virtio_net --vcpu=4 --testcase=virtio_fs_share_data..run_stress.with_xfstest..with_cache.none --netdst=virbr0 --debug --clone=no
```
2023-10-12 01:47:52: [  332.683039] run fstests generic/028 at 2023-10-12 13:47:52
2023-10-12 01:47:58: [  337.940461] run fstests generic/031 at 2023-10-12 13:47:57
```
```
2023-10-12 02:04:43: [ 1342.801572] run fstests generic/245 at 2023-10-12 14:04:42
2023-10-12 02:04:43: [ 1343.066995] run fstests generic/249 at 2023-10-12 14:04:42
```
python3 ConfigTest.py --guestname=RHEL.9.4.0..page_64k --platform=aarch64 --machines=arm64-pci --driveformat=virtio_scsi --nicmodel=virtio_net --vcpu=4 --testcase=virtio_fs_share_data..run_stress.with_xfstest..with_cache.auto --netdst=virbr0 --debug --clone=no
```
2023-10-12 03:25:28: [  302.117568] run fstests generic/028 at 2023-10-12 15:25:27
2023-10-12 03:25:33: [  307.336990] run fstests generic/029 at 2023-10-12 15:25:33
2023-10-12 03:25:33: [  307.672492] run fstests generic/030 at 2023-10-12 15:25:33
2023-10-12 03:25:34: [  308.510178] run fstests generic/031 at 2023-10-12 15:25:34
```
```
2023-10-12 03:53:24: [ 1978.126843] run fstests generic/245 at 2023-10-12 15:53:23
2023-10-12 03:53:24: [ 1978.365500] run fstests generic/246 at 2023-10-12 15:53:24
2023-10-12 03:53:24: [ 1978.608385] run fstests generic/247 at 2023-10-12 15:53:24
2023-10-12 03:53:30: [ 1984.346694] run fstests generic/248 at 2023-10-12 15:53:30
2023-10-12 03:53:30: [ 1984.583445] run fstests generic/249 at 2023-10-12 15:53:30
```
